menu/server_configuration.sh

```diff
@@ -76,12 +74,18 @@ https://www.techandme.se/open-port-80-443/"
             print_text_in_color "$ICyan" "OK, but if you want to run it later, just type: sudo bash $SCRIPTS/activate-tls.sh"
             any_key "Press any key to continue..."
         fi
-
-        # Just make sure they are gone
-        rm -f "$SCRIPTS/test-new-config.sh"
```

Comment:
Please add back rm for the test script.

Thanks!

Reply:
done 👍